test/e2e/framework/ports.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I was wondering we could define this file as e2e sub framework.
But that would make invalid dependency from core framework to the sub framework because  test/e2e/framework/resource_usage_gatherer.go and util.go are using already.
So it is fine to merge as this PR.